### PR TITLE
support networking.k8s.io/v1 ingress api

### PR DESCRIPTION
**Pull request by jodybro**
## Hey, I just made a Pull Request!

This PR adds a conditional check for the ingress object that checks the version of the target k8s cluster and sets the api type to 
`networking.k8s.io/v1beta1` when the minor k8s version is less than 19 and `v1` for greater than 19.

#### :heavy_check_mark: Checklist

- [ ] A changeset describing the change and affected packages. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#creating-changesets))
- [ ] Added or updated documentation
- [ ] Tests for new functionality and regression tests for bug fixes
- [ ] Screenshots attached (for UI changes)
- [ ] All your commits have a `Signed-off-by` line in the message. ([more info](https://github.com/backstage/backstage/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))

### CODE REVIEW

**Comment by benjdlambert** [comment]
Hey @jodybro, we need to sign off the commit with a DCO check. You can do `git commit --amend --signoff` and push the branch again to fix :)

**Comment by jodybro** [comment]
> Hey @jodybro, we need to sign off the commit with a DCO check. You can do `git commit --amend --signoff` and push the branch again to fix :)

Gotcha!

File: contrib/chart/backstage/templates/ingress.yaml [hunk 1]
```diff
@@ -1,7 +1,13 @@
 {{- $frontendUrl := urlParse .Values.appConfig.app.baseUrl}}
 {{- $backendUrl := urlParse .Values.appConfig.backend.baseUrl}}
 {{- $lighthouseUrl := urlParse .Values.appConfig.lighthouse.baseUrl}}
+
+{{/* Determine the api type for the ingress */}}
+{{- if lt .Capabilities.KubeVersion.Minor "19" }}
 apiVersion: networking.k8s.io/v1beta1
+{{- else if gt .Capabilities.KubeVersion.Minor "19" }}
```
Comment (freben) [inline contrib/chart/backstage/templates/ingress.yaml:8]: wait, does this really work when it's _exactly_ 19? :) isn't there a plain `else` to use?

**Comment by jodybro** [comment]
Ahh damn I didnt branch off before creating this PR. Ill just remake it rather than dealing with git magic. My mistake, sorry 😢

**Pull request closed without merging**